src/garage/experiment/local_runner.py

```diff
@@ -148,6 +148,9 @@ def __init__(self, snapshot_config, max_cpus=1):
         self.step_itr = None
         self.step_path = None
 
+        self._n_workers = None
```

Comment:
can you put these changes to LocalRunner in a separate PR so that @naeioi @zequnyu and @krzentner can review them?

Reply:
I'm already pretty familiar with these changes, so I'm fine reviewing them here. I think they're fine.